test/e2e_node/restart_test.go

```diff
@@ -64,7 +64,7 @@ func waitForPods(f *framework.Framework, podCount int, timeout time.Duration) (r
 	return runningPods
 }
 
-var _ = SIGDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
+var _ = SIGDescribe("Restart [Serial] [Slow] [Disruptive] [Flaky]", func() {
```

Comment:
Can you move this to L87? `ginkgo.It("should recover from ip leak",` -- there's another test in here for restarting Kubelet that SIG Describe will mark as flaky when it's not

Reply:
Done.